test/e2e/empty.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"time"
+
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = framework.KubeDescribe("[Feature:Empty]", func() {
+	f := framework.NewDefaultFramework("empty")
+
+	BeforeEach(func() {
```

Comment:
Please add a TODO to incorporate something like this:
https://github.com/kubernetes/kubernetes/pull/32108

into those two functions that you are using(WaitForAllNodesSchedulable and WaitForAllNodesHealthy).